src/main/java/uk/gov/pay/api/service/GetPaymentRefundsService.java

```diff
@@ -1,20 +1,64 @@
 package uk.gov.pay.api.service;
 
+import uk.gov.pay.api.app.config.PublicApiConfig;
 import uk.gov.pay.api.auth.Account;
+import uk.gov.pay.api.model.Refund;
+import uk.gov.pay.api.model.RefundResponse;
 import uk.gov.pay.api.model.RefundsResponse;
 
 import javax.inject.Inject;
+import javax.ws.rs.core.UriBuilder;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static uk.gov.pay.api.resources.PaymentRefundsResource.PAYMENT_BY_ID_PATH;
+import static uk.gov.pay.api.resources.PaymentRefundsResource.PAYMENT_REFUNDS_PATH;
+import static uk.gov.pay.api.resources.PaymentRefundsResource.PAYMENT_REFUND_BY_ID_PATH;
```

Comment:
I don't like the fact that service that is used by the resource imports static things from this resource (sounds like circular dependency to me). Should be moved somewhere else (follow up PR I guess)

Reply:
Good spot! will replace in next PR